File: backend/Entities/Models/Transportador1.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class Transportador1
{
    public Guid Id { get; set; }

    public Guid TenantId { get; set; }

    public Guid TransportadorId { get; set; }

    public int? TipoContaId { get; set; }

    public string? ChavePix { get; set; }

    public string? Banco { get; set; }

    public string? Agencia { get; set; }

    public string? AgencidaDigito { get; set; }

    public string? Conta { get; set; }

    public string? ContaDigito { get; set; }

    public string? CnpjCpfFavorecido { get; set; }

    public string? NomeFavorecido { get; set; }

    public virtual Pessoa IdNavigation { get; set; } = null!;

    public virtual Tenant Tenant { get; set; } = null!;

    public virtual TipoContum? TipoConta { get; set; }

    public virtual Transportador Transportador { get; set; } = null!;
}

File: frontend-with-menus/src/app/Business/Models/transportador1.ts
import { Pessoa } from "./pessoa";
import { Tenant } from "./tenant";
import { TipoContum } from "./tipoContum";
import { Transportador } from "./transportador";


export interface Transportador1  {
    id: string;
    transportadorId: string;
    tenantId: string;
    titularId: string;
    tipoContaId: number | null;
    chavePix: string | null;
    banco: string | null;
    agencia: string | null;
    agencidaDigito: string | null;
    conta: string | null;
    contaDigito: string | null;
    cnpjCpfFavorecido: string | null;
    nomeFavorecido: string | null;
    idNavigation: Pessoa;
    tenant: Tenant;
    tipoConta: TipoContum | null;
    transportador: Transportador;
}